packages/editor/templates/Crypto Pizzas/contracts/CryptoPizza.sol

```diff
@@ -1,89 +1,107 @@
+// Specifies that the source code is for a version
+// of Solidity greater than 0.5.10
 pragma solidity ^0.5.10;
 
-import "../node_modules/openzeppelin-solidity/contracts/token/ERC721/IERC721.sol";
-import "../node_modules/openzeppelin-solidity/contracts/token/ERC721/IERC721Receiver.sol";
-import "../node_modules/openzeppelin-solidity/contracts/introspection/ERC165.sol";
-import "../node_modules/openzeppelin-solidity/contracts/math/SafeMath.sol";
+// Imports symbols from other files into the current contract.
```

Comment:
I recommend adding a reference here (e.g. https://solidity.readthedocs.io/en/v0.6.2/layout-of-source-files.html#importing-other-source-files). I wasn't aware of what "symbols" meant... 😁

Reply:
Kind of in the main read me, as then the file gets messy, an you can't click the links either, but for this particular one being so fundamental, done!